ministryofjustice/correspondence_tool_staff
# CT-3037, CT-3009 Performance percentage and report improvement

## Description

This PR is to provide the improvement around monthly performance report 
- Removed redundant fields for offender sar monthly report
- Give different performance percentage calculation for different case type
- Add extra notes for explaining the report


possible improvement in the future is to templatise the report and place the formula in side xlsx template ( maybe other format as well)
## Self-review checklist


### Screenshots

Before
<img width="644" alt="Screenshot 2020-10-01 at 12 49 51" src="https://user-images.githubusercontent.com/26111671/94805669-acb67e80-03e4-11eb-9790-a2f4fe1e0c04.png">


After
Offender SAR
<img width="1053" alt="Screenshot 2020-10-01 at 12 44 57" src="https://user-images.githubusercontent.com/26111671/94805266-04a0b580-03e4-11eb-9dbb-b63d22693ad1.png">
FOI etc
<img width="717" alt="Screenshot 2020-10-01 at 12 44 35" src="https://user-images.githubusercontent.com/26111671/94805285-0c605a00-03e4-11eb-95cf-d797d0c6feed.png">

### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3037
https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3009
### Deployment


### Manual testing instructions

FOI, SAR etc
- Login as London user
- Click "Report"
- Click to download FOI - "Monthly performance report""
- Check the data 

Offender SAR
- Login as London user
- Click "Report"
- Click to download Offender SAR - "Monthly performance report""
- Check the data
